extensions/interactions/ImageClickInput/directives/ImageClickInput.js

```diff
@@ -59,23 +63,55 @@ oppia.directive('oppiaInteractiveImageClickInput', [
             if ($scope.dimensions.height < 124) {
               loadingIndicatorSize = 24;
             }
-            var paddingTop = Math.max(0, (($scope.dimensions.height * 0.5) -
-              (loadingIndicatorSize * 0.5)));
+
+            var paddingTopForLoadingIndicator = Math.max(0, ((
+              $scope.dimensions.height * 0.5) - (loadingIndicatorSize * 0.5)));
             $scope.loadingIndicatorContainerStyle =
             {
-              'padding-top': paddingTop + 'px',
+              'padding-top': paddingTopForLoadingIndicator + 'px',
```

Comment:
If you add to `oppia-image-container` CSS
```css
  display: flex;
  align-items: center;
```
You don't need to calculate the padding. The same then applies to try again button.

Reply:
Done!

Reply:
Here is the screenshot displaying the try again button
![tryagain](https://user-images.githubusercontent.com/23234392/41375423-ad7f520e-6f73-11e8-8208-8887763acf15.png)

Reply:
Nice, thanks!